docs/articles_en/openvino_workflow/model_optimization_guide/weight_compression.md

```diff
@@ -28,6 +30,32 @@ The code snippet below shows how to compress the weights of the model represente
 
 Now, the model is ready for compilation and inference. It can be also saved into a compressed format, resulting in a smaller binary file.
 
+- **INT4 weight quantization** - this method stands for an INT4-INT8 mixed-precision weight quantization where INT4 is considered as a primary precision and INT8 is a backup one. It usually results in a smaller model size and a lower inference latency but accuracy degradation is also higher. The method has several parameters that can provide different performance-accuracy trade-offs after optimization:
+
+  * `mode` - there are two modes to choose from: `INT4_SYM` - stands for INT4 symmetric weight quantization and results in faster inference and smaller model size, and `INT4_ASYM` - INT4 asymmetric weight quantization with variable zero-point for more accurate results
+
+  * `group_size` - controls the size of the group of weights that share the same quantization parameters. The smaller the model size the more accurate the optimized model. We recommend using the following group sizes: `128`, `64`, `32` (`128` is default value)
+
+  * `ratio` - controls the ratio between INT4 and INT8 compressed layers in the model. For example, 0.8 means that 80% of layers will be compressed to INT4 while the rest to INT8 precision.
+
```

Comment:
Could we add some statement about selection of those parameters and (if reasonable) could we maybe make table with most popular models and recommended settings?
Like this: 
Please see below few examples of known to be working well settings for most popular models:
<hf link> <sym/asym> <gsize> <ratio>
This should be easy to get, but could also indicate which models are supported

Reply:
@ljaljushkin, do we have such recommendations that we can place here?

Reply:
Here's the best configuration for a few models that I found so far: 
https://github.com/openvinotoolkit/nncf/blob/abce5d9bc126786844a9c2599ab6e0ad04ba9dff/docs/compression_algorithms/CompressWeights.md#evaluation-results
We could extend it for other models. 
Regarding recommendations, we know how each separate parameter influence on accuracy and performance, and it's described in this PR.
But it's hard to say how would they work in combination for each individual model. Evaluation of accuracy and performance of many combinations on a big scope of models should give some insights.

Reply:
Let's put at least a link to that NNCF documentation here!

Reply:
@ngaloppo, I don't think this is the right way since we have also results for the experimental NF4 type and they are much better than INT4. It can confuse users because NF4 works only for AVX-512 + FP32 inference (i.e. no support of SPR and new client CPUs or any GPU). It is better to extract this table and place it here.

Reply:
Updated

Reply:
Got it, thanks for clarifying!